### ISSUE TITLE
Horizontal scrolling broken in 0.32.0-rc.0

### ISSUE THREAD

**Issue by rozele** [ReactWindows/react-native-windows#561]
Horizontal scrolling example is broken in UIExplorer.

**Comment by rozele** [comment]
Confirmed this is also an issue for 0.31.0.

**Comment by rozele** [comment]
Turns out this is related to the update to Facebook.CSSLayout v2.0.1-pre.  We may need to revert https://github.com/ReactWindows/react-native-windows/commit/104aa49fc403fd6b05aab474baee90235f8e2c99 (cc @rigdern)

**Comment by rozele** [comment]
@rigdern, it looks like if you call CalculateLayout() on the horizontal scrolling content, it works okay, but I'm not sure the right infrastructure for that.  In the meantime, I'm going to revert  https://github.com/ReactWindows/react-native-windows/commit/104aa49fc403fd6b05aab474baee90235f8e2c99 and we can work on integrating it in a future release.

**Comment by rigdern** [comment]
@rozele I can take a look at this later today.

Just to double check, are you using the same UIExplorer code used in the iOS and Android versions of 0.31.0? After updating the layout engine, Facebook had to make some tweaks to the UIExplorer app to fix bugs due to differences in the layout engine implementation. For example, facebook/react-native#8248 was introduced and was fixed by https://github.com/facebook/react-native/commit/dc5b4903dd87d28910beb67ea74829900afdad48.

**Comment by rigdern** [comment]
@rozele What's the latest news on this? I saw that this bug got closed without a corresponding bug being opened for upgrading the layout engine. Should I still look into this or did you figure it out? If I need to look into it, are there instructions somewhere for running the UIExplorer app? Also, I'm still curious to hear an answer to the question in my earlier comment.

**Comment by rozele** [comment]
I didn't open a new bug yet, but, yes, this still needs to be investigated.  I think there's a README in the Examples folder for how to run the UIExplorer (or it may be the main README).

Sent from my Windows Phone
________________________________

**Comment by rigdern** [comment]
I got UIExplorer running, diagnosed, and fixed the isse (PR #565).

The reason I was confused about how to run UIExplorer was I didn't see it in my local repo so I assumed it lived somewhere else. [GitHub's file search](https://github.com/ReactWindows/react-native-windows/find/master) also turned up 0 results for "UIExplorer". I didn't realize there was a git submodule I had to initialize to get UIExplorer.
